android-refimpl-app/app/src/main/res/values/strings.xml

```diff
@@ -195,6 +195,10 @@
     QR code button to scan friend\'s code with a camera.</string>
     <string name="add_friend_activity_add">Add Friend</string>
     <string name="cancel">Cancel</string>
+
+    <string name="friend_request_sent"> Friend request has been sent.</string>
```

Comment:
It just looks nicer in the app. It is actually not used, the visibility of text view using it is gone since I tried a couple of layouts, and found that a green yes icon is most intuitive to users. I simply forgot to delete the unused text view and this string. Sorry for the confusstion.